ringpop.thrift-gen

```diff
@@ -0,0 +1,170 @@
+{{ define "arguments" }}ctx {{ contextType }}{{ range .Arguments }}, {{ .Name }} {{ .Type | goType }}{{ end }}{{ end}}
```

Comment:
Can you drop in an example of the generated code somewhere? It'll be nice to review that side of the coin too.